cli/command/service.go

```diff
@@ -10,6 +16,260 @@ func NewServiceCommand(b *CommandBase) *ServiceCommand {
 	return &ServiceCommand{b}
 }
 
-func (e *ServiceCommand) Command() cli.Command {
-	return cli.Command{}
+func (s *ServiceCommand) Command() cli.Command {
+	return cli.Command{
+		Name:  "service",
+		Usage: "manage Layer0 services",
+		Subcommands: []cli.Command{
+			{
+				Name:      "create",
+				Usage:     "create a new service",
+				Action:    s.create,
+				ArgsUsage: "ENVIRONMENT NAME DEPLOY",
+				Flags: []cli.Flag{
+					cli.StringFlag{
+						Name:  "loadbalancer",
+						Usage: "attach the service to the specified load balancer",
+					},
+				},
+			},
+			{
+				Name:      "delete",
+				Usage:     "delete a service",
+				Action:    s.delete,
+				ArgsUsage: "NAME",
+			},
+			{
+				Name:      "get",
+				Usage:     "describe a service",
+				Action:    s.read,
+				ArgsUsage: "NAME",
+			},
+			{
+				Name:   "list",
+				Usage:  "list all services",
+				Action: s.list,
+			},
+			{
+				Name:      "logs",
+				Usage:     "get the logs for a service",
+				Action:    s.logs,
+				ArgsUsage: "NAME",
+				Flags: []cli.Flag{
+					cli.IntFlag{
+						Name:  "tail",
+						Usage: "number of lines from the end to return (default: 0)",
+					},
+					cli.StringFlag{
+						Name:  "start",
+						Usage: "the start of the time range to fetch logs (format: YYYY-MM-DD HH:MM)",
+					},
+					cli.StringFlag{
+						Name:  "end",
+						Usage: "the end of the time range to fetch logs (format: YYYY-MM-DD HH:MM)",
+					},
+				},
+			},
+			{
+				Name:      "scale",
+				Usage:     "scale a service",
+				Action:    s.scale,
+				ArgsUsage: "NAME COUNT",
+			},
+			{
+				Name:      "update",
+				Usage:     "run a new dploy on a service",
+				Action:    s.update,
+				ArgsUsage: "NAME DEPLOY",
+			},
+		},
+	}
+}
+
+func (s *ServiceCommand) create(c *cli.Context) error {
+	args, err := extractArgs(c.Args(), "ENVIRONMENT", "NAME", "DEPLOY")
+	if err != nil {
+		return err
+	}
+
+	deployID, err := s.resolveSingleEntityIDHelper("deploy", args["DEPLOY"])
+	if err != nil {
+		return err
+	}
+
+	environmentID, err := s.resolveSingleEntityIDHelper("environment", args["ENVIRONMENT"])
+	if err != nil {
+		return err
+	}
+
+	var loadBalancerID string
+	if loadBalancerName := c.String("loadbalancer"); loadBalancerName != "" {
+		id, err := s.resolveSingleEntityIDHelper("load_balancer", loadBalancerName)
+		if err != nil {
+			return err
+		}
+
+		loadBalancerID = id
+	}
+
+	req := models.CreateServiceRequest{
+		DeployID:       deployID,
+		EnvironmentID:  environmentID,
+		LoadBalancerID: loadBalancerID,
+		ServiceName:    args["NAME"],
+	}
+
+	jobID, err := s.client.CreateService(req)
+	if err != nil {
+		return err
+	}
+
+	onCompleteFN := func(serviceID string) error {
+		service, err := s.client.ReadService(serviceID)
```

Comment:
We need to add back in the `WaitForDeployment` helper function from develop and do that here. 
I think it should be another non-interface helper like [WaitForJob](https://github.com/quintilesims/layer0/blob/cli-refactor/client/helpers.go#L33)

Reply:
✅